### DISCUSSION TITLE
Scope of Observer

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1874] [Questions & Problems] [answered]
I created a container observer that shows the total amount of Itraconazole in the small intestine mucosa intracellular compartments that has come in from the corresponding lumen compartments. 

![Screenshot 2024-12-04 194932](https://github.com/user-attachments/assets/6b31ad92-64a6-4aff-8f65-33bcfea35488)

Can I expand the scope of this observer to include molecules other than Itraconazole? Maybe by changing the reference paths from: 

"Lumen_duo_Duodenum_cell|Itraconazole|Drug absorption rate (lumen to mucosa)" to something like "Lumen_duo_Duodenum_cell|**|Drug absorption rate (lumen to mucosa)"?
 
Or, do I have to create a new container observer for every other molecule I am interested in? 

Lastly, is it possible to do all, or any, of this in R?

**Comment by PavelBal** [comment, marked as answer]
You can use the keyword `MOLECULE`, so `Lumen_duo_Duodenum_cell|MOLECULE|Drug absorption rate (lumen to mucosa)`.

Unfortunately, you cannot create observers in R.

**Reply by DonkeySaddle** [reply]
Thanks @PavelBal this is great.

**Reply by DonkeySaddle** [reply]
@PavelBal, long shot, but I assume you can't load existing observer pkml files into R and add them onto the simulation you're working with either?

**Reply by PavelBal** [reply]
No, right now you cannot combine/create simulations from R, only load those created in PK-Sim/MoBi. The feature is planned for v13, though.

**Reply by DonkeySaddle** [reply]
Looking forward to it! Thanks @PavelBal
